kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/QuizResourceSelection/index.vue

```diff
@@ -488,14 +488,16 @@
         if (selectPracticeQuiz) {
           return !workingPoolHasChanged.value;
         }
-        return (
-          !workingPoolHasChanged.value ||
-          (!settings.value.isChoosingManually &&
-            workingPoolQuestionsCount.value < settings.value.questionCount) ||
-          settings.value.questionCount < 1 ||
-          tooManyQuestions.value ||
-          settings.value.questionCount.value > settings.value.maxQuestions
-        );
+        const disabledConditions = [
```

Comment:
this is not blocking, and is a bit of a higher level reflection, but I think many of the changes in the quiz workflow added over the last few months make perfect sense within the context of their own PR, and also as we have gone along adding more conditions and options building out these features, these files are starting to get a bit unwieldy. Especially with some of the underlying complexity from the 0.17 work. 

I'd like us to think about some small, incremental refactors that we can do over the course of our patch releases of 0.18. If you have ideas in mind already, Alex, let's start making note of them :)